lesson_09/homework_09.py

```diff
@@ -0,0 +1,79 @@
+import string
+
+class CustomErrorClass(Exception):
+    def __init__(self, *, message, value):
+        super().__init__()
+        self.exception = f"{message}, current instance: '{value}'"
+
+
+def multiplication_table(number: [int, float]):
+    # Initialize the appropriate variable
+    multiplier = 1
+    result_array = []
+    try:
+        if not isinstance(number, (int, float)):
+            raise CustomErrorClass(message="Instance must be 'int' or 'float' type", value=number.__class__.__name__)
+        # Complete the while loop condition. Loop starts with positive number only.
+        while number > 0:
```

Comment:
```suggestion
        while True:
```
здається, різниці не буде

Reply:
Тоді умова "result > 25" може стати недосяжна і цикл стане нескінченим. Переписав логіку функції і негативний тест став непотрібний. Функція або працює або райзіть помилки.